### DISCUSSION TITLE
Cursor not changing

### DISCUSSION THREAD

**Discussion by ghost** [xournalpp/xournalpp#3053] [Q&A] [answered]
I have switched from a Wacom tablet to an XP-Pen Artist 15.6 Pro pen-display. Now the cursor does not change from arrow to point when moving over the drawing area. Also true for highlighter. When using the eraser, the cursor changes immediately an stays as eraser rectangle all the time.
For pen the cursor only changes when touching the display, but is back to mouse arrow when removing from display. This is not the case for example when using krita, gimp, inkscape - so I assume not a driver problem but a Xournalpp problem.

Any idea?

**Comment by rolandlo** [comment, marked as answer]
In case you use [version 1.1.0+dev](https://github.com/xournalpp/xournalpp/releases/tag/nightly) you should get the desired behavior in case your pen is correctly configured as pen (and not as a mouse) in the input system panel in menu `Edit > Preferences`.

**Reply by ghost** [reply]
Works now - thank you !
